### ISSUE TITLE
Kleurverschil in status en legende

### ISSUE THREAD

**Issue by CatoCoordinator** [inbo/vespa-db#127]
Nesten hebben een legende (bolletje) gebaseerd op gerapporteerd (zwart), gereserveerd (geel) en bestreden (groen).

Wanneer je een nest reserveert als ingelogde bestrijder, krijgt dit bolletje de kleur **oranje op de kaart**. Deze kleur is net iets donkerder dan in de **legende (geel)**.

Wanneer je nadien een ander nest aanduid, veranderd het **gereserveerde nest (oranje/geel bolletje)** terug naar een **gerapporteerd nest (zwart bolletje).** Dit zou dan eigenlijk de originele kleur van gereserveerd nest (geel zoals legende) moeten hebben.

**Comment by peterdesmet** [comment]
Inderdaad, er is nog steeds een verschil, wat te wijten is aan `--v-theme-warning` vs `--bs-warning-rgb`. Het laatste is de kleur van de legende en kan best overal gebruikt worden, aangezien het oranje kleur voor "geselecteerd" wordt gebruikt:

<img width="902" alt="Screenshot 2024-07-16 at 11 46 43" src="https://github.com/user-attachments/assets/31b6d7f7-d9b9-41a1-8e43-62ef9d2995b2">

**Comment by peterdesmet** [comment]
Idem voor bestreden, een verschil tussen `--v-theme-success` en `--bs-success-rgb`

**Comment by peterdesmet** [comment]
@CatoCoordinator het kleurverschil `--v-theme-warning` vs `--bs-warning-rgb` en `--v-theme-success` en `--bs-success-rgb` is er nog steeds, dus ik zet deze issue terug open.

**Comment by jrhillae** [comment]
@peterdesmet:  de oplossing was normaal om gereserveerd aan te duiden met geel en geselecteerd oranje? Momenteel is alles oranje. Best aanpassen maar volgens mij wel geen hoge prioriteit want er is volgens mij weinig verwarring als gebruiker tss geselecteerd en gereserveerd omdat bij geselecteerd enkel de rand kleurt en niet het bolletje zelf. Mag ook later op het jaar worden opgelost.
![image](https://github.com/user-attachments/assets/91c5cc9d-ccd4-4d6a-a907-ffd3dab2bb8c) @PietrH : testing failed

**Comment by jrhillae** [comment]
Nog eens opnieuw getest dag later en nu lijkt de kleur anders:
![image](https://github.com/user-attachments/assets/0be5f315-5214-4b3f-a132-6884b0fd8207)
